spec/cypress/end_to_end/research_plan_with_user.cy.js

```diff
@@ -13,4 +13,36 @@ describe('Research Plan with User', () => {
     cy.contains('My Research Plan 2');
     cy.get('#tabList-tab-4 > span').contains('1(0)');
   });
+
+  it('rename research plan', () => {
```

Comment:
Why are there two tests called `rename research plan` (the other one on line 2)?